packages/ui-library/src/components/input-field-text/index.css.ts

```diff
@@ -86,10 +86,11 @@ export const styleCustom = typeSafeNestedCss/*css*/ `
             }
           }
 
-          &.disabled {
+          &[disabled] {
```

Comment:
sorry for my earlier advice but i think we should change this back to a class selector since other styles seem to depend on this as well. see line 206. there also seems to be a color definition for the blr-input-wrapper in that line.

Reply:
I have reverted this thanks for pointing out.